libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/api/MapboxRouteLineApi.kt

```diff
@@ -221,7 +223,7 @@ class MapboxRouteLineApi(
         newRoutes: List<RouteLine>,
         consumer: MapboxNavigationConsumer<Expected<RouteLineError, RouteSetValue>>
     ) {
-        jobControl.scope.launch {
+        jobControl.scope.launch(Dispatchers.Main) {
```

Comment:
The idea is to provide a thread pool separate from ThreadController. Using `Main` doesn't seem that useful to me since it's only 1 thread and the idea is to get tasks off of the main thread.  Using `Default` utilizes the default JVM thread pool of at least 2 threads without creating a new thread pool as ThreadController does.

Reply:
Cool that makes sense. Thanks for clarifying 🙇‍♂️